llvm/include/llvm/ProfileData/InstrProf.h

```diff
@@ -605,6 +703,19 @@ Function* InstrProfSymtab::getFunction(uint64_t FuncMD5Hash) {
   return nullptr;
 }
 
+GlobalVariable *
+InstrProfSymtab::getGlobalVariable(uint64_t GlobalVariableMD5Hash) {
+  finalizeSymtab();
```

Comment:
Why do we need to finalizeSymtab when looking up a global var?
Also just using a DenseMap for `MD5VTableMap` means this code can be simplified to a one liner probably.